pkgs/development/python-modules/x256/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "x256";
+  version = "0.0.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "00g02b9a6jsl377xb5fmxvkjff3lalw21n430a4zalqyv76dnmgq";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Find the nearest xterm 256 color index for an RGB";
+    homepage = https://github.com/magarcia/python-x256;
+    license = licenses.mit;
+    maintainers = with maintainers; [ Scriptkiddi ];
+    platforms = platforms.unix;
```

Comment:
Unless there is a reason to, `platforms` should not be set for `buildPython*`. It will be set automatically.

Reply:
```suggestion
```